pallets/usdv-minting/Cargo.toml

```diff
@@ -9,6 +9,7 @@ valiu-node-commons = { default-features = false, path = '../../valiu-node-common
 
 [dev-dependencies]
 orml-tokens = { default-features = false, version = '0.3' }
+pallet-provider = { default-features = false, path = '../liquidity-provider' }
```

Comment:
This got me thinking about what you mentioned at some point of having a single pallet instead of splitting into two. Having to use the provider pallet like this is a design smell, probably the two things belong together. With the way we are going to follow it probably makes sense to make the minting part of what a provider does, to keep both decoupled it would require that attested assets go to a central pot(some configurable accountId) where the "minters" would take the collateral to create USDv.  
Anyway sorry that the design has been a bit uncertain but now I have more clear that USDv creation should be part of the functions of the liquidity provider and actually I think it should be done automatically as soon as a provider makes an attestation with a "collateral asset".

Imagining it can be something like [this](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=0646cc3f7e4edfddd4ebf88f2620ce1b)